docs/v2/api-endpoints/sample_mixins.md

```diff
@@ -8,7 +8,8 @@ description: Sample a desired number of mixins from the ledger, excluding a list
 
 | Required Param | Purpose | Requirements |
 | :--- | :--- | :--- |
-| `outputs` | The TXOs to get the membership proofs for | TXO must exist in the ledger |
+| `num_mixins` | ??? | ??? |
+| `excluded_outputs` | ??? | ??? |
```

Comment:
what should be the proper Purpose and Requirements for these two parameters?

Reply:
| `num_mixins` | The number of mixins to sample | Must be less than the number of txos in the ledger minus number of excluded outputs |
| `excluded_outputs` | Txos to exclude from sampling | Txo must exist in the ledger |

Reply:
got it, thank you!